web/styles/zulip.css

```diff
@@ -1866,6 +1868,15 @@ nav {
                 height: 15px;
             }
         }
+
+        .left-sidebar-toggle-button {
+            margin-right: 5px;
+
+            .left-sidebar-toggle-unreadcount {
+                top: 10px;
+                right: 14px;
+            }
```

Comment:
Can we do less fragile positioning for this element using grid of flexbox?

Reply:
We could theoretically but likely not worth effort of figuring out how to get the same position using flex/grid.

Reply:
Actually, we should just have two versions of this icon and get them rendered.

Reply:
https://chat.zulip.org/#narrow/stream/101-design/topic/left.20sidebar.20toggle.20icon